app/controllers/claims_controller.rb

```diff
@@ -4,11 +4,20 @@ class ClaimsController < ApplicationController
 
   after_action :clear_claim_session, if: :submission_complete?
 
+  def new
+    @current_claim = Claim.new(eligibility: StudentLoans::Eligibility.new)
+    render "qts_year"
+  end
+
   def create
-    claim = Claim.create!(eligibility: StudentLoans::Eligibility.new)
-    session[:claim_id] = claim.to_param
+    @current_claim = Claim.new(eligibility: StudentLoans::Eligibility.new)
 
-    redirect_to claim_path("qts-year")
+    if update_current_claim!
+      session[:claim_id] = @current_claim.to_param
+      redirect_to claim_path(next_slug)
+    else
+      render "qts_year"
```

Comment:
as above with `new`, if we can abstract this away so that the controller action isn't coupled to the string for the first page in the sequence it would be better.

Reply:
Yep, now we have access to `current_claim`, we can use `claim_page_template`